ivy_tests/test_ivy/test_frontends/test_tensorflow/test_raw_ops.py

```diff
@@ -4345,3 +4345,32 @@ def test_tensorflow_Svd(
         atol=1e-2,
         ground_truth_backend=frontend,
     )
+
+
+@handle_frontend_test(
+    fn_tree="tensorflow.raw_ops.Selu",
+    dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("float"),
+        min_num_dims=1,
+    ),
+    test_with_out=st.just(False),
+)
+def test_tensorflow_Selu(
+    *,
+    dtype_and_x,
+    frontend,
+    test_flags,
+    fn_tree,
+    backend_fw,
+    on_device,
+):
+    dtype, x = dtype_and_x
+    helpers.test_frontend_function(
+        input_dtypes=dtype,
+        backend_to_test=backend_fw,
+        frontend=frontend,
+        test_flags=test_flags,
+        fn_tree=fn_tree,
+        on_device=on_device,
```

Comment:
You can add `rtol=1e-2` and `atol=1e-2` to remove failing checks due to small rounding errors.